projects/ngrx.io/content/guide/store/why.md

```diff
@@ -19,31 +19,31 @@ A good substance that might answer the question "Do I need NgRx", is the
 
 * **I**mpacted: state that is impacted by actions from other sources.
 
-However, realizing that using NgRx comes with some tradeoffs is also crucial. It is not meant to be the shortest or quickest way to write code and encourage its users the usage of many files.
+However, realizing that using NgRx comes with some tradeoffs is also crucial. It is not meant to be the shortest or quickest way to write code. It also encourages the usage of many files.
 
 It's also important to consider the patterns implemented with NgRx Store. A solid understanding of [`RxJS`](https://rxjs.dev) and [`Redux`](https://redux.js.org/) will be very beneficial before learning to use NgRx Store and the other state management libraries.
 
 ## Key Concepts
 
 ### Type Safety
 
-Type safety is promoted throughout the architecture with reliance on the TypeScript compiler for program correctness.
+Type safety is promoted throughout the architecture via the use of TypeScript.
```

Comment:
Thanks for all the suggestions! 👍

This one is an interesting one. All state management libs for Angular use TS, however not all are as strict with it, or use the patterns that would help maximize the type-safety. Not sure how to word it better 🙂

Reply:
Glad to help!

I could add a sentence about it after this. Perhaps it could read:

Type safety is promoted throughout the architecture via the use of TypeScript. In addition to this, NgRX's strictness of type-safety and use of patterns lends itself well to the creation of higher quality code.

Would this be better?

Reply:
That sound good 👍 😀

Reply:
That's what I get for typing too fast. :) I went ahead and updated it.

Reply:
Hi Alex, should I mark this as resolved or does the reviewer typically do it?